.teamcity/settings.kts

```diff
@@ -72,7 +72,7 @@ fun Project.buildVersion() = BuildType {
     commonConfigure()
 
     params {
-        param(versionSuffixParameter, "SNAPSHOT")
+        param(versionSuffixParameter, "")
```

Comment:
Is it possible to have this preserved when the settings are regenerated after the next kotlinx.team.infra plugin update?

Reply:
This change is now moved to `additionalConfiguration.kt`. Thanks!